tests/integration/test_core.py

```diff
@@ -126,61 +123,16 @@ async def test_openstack_object_storage_plugin(
         After openstack swift plugin activated, an image file uploaded to one unit through
         WordPress media uploader should be accessible from all units.
     """
+    if swift_config is None:
```

Comment:
Would it be possible to use this? https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-mark-skipif

Reply:
To skip the test or not depends on a fixture value. `pytest.mark.skipif` does not support that I think.

Reply:
Hmm, how often would this be skipped and what would be the reason for the skip? Is this if a parameter isn't supplied or something like that?

Reply:
The WordPress charm support two mode: connecting to an external database using charm config and connecting to a charmed database using charm relation. The skip here is to enable integration tests to test both cases with different command line parameters.